windows/internal/cuda_install.bat

```diff
@@ -182,7 +184,9 @@ if not exist "%SRC_DIR%\temp_build\%CUDA_INSTALL_EXE%" (
     set "ARGS=thrust_11.5 nvcc_11.5 cuobjdump_11.5 nvprune_11.5 nvprof_11.5 cupti_11.5 cublas_11.5 cublas_dev_11.5 cudart_11.5 cufft_11.5 cufft_dev_11.5 curand_11.5 curand_dev_11.5 cusolver_11.5 cusolver_dev_11.5 cusparse_11.5 cusparse_dev_11.5 npp_11.5 npp_dev_11.5 nvrtc_11.5 nvrtc_dev_11.5 nvml_dev_11.5"
 )
 
-set CUDNN_INSTALL_ZIP=cudnn-11.3-windows-x64-v8.2.0.53.zip
+set CUDNN_FOLDER=cudnn-windows-x86_64-8.3.2.44_cuda11.5-archive
+Set CUDNN_LIB_FOLDER="lib"
```

Comment:
is there a diff between capital Set and lowercase set?

Reply:
Sorry don't understand the question. Nvidia changed the way they package CUDNN and instead of CUDNN being extracted to cuda folder they extract it now into archive name folder. And also lib folder was changed.

Reply:
Oh i just meant 
```suggestion
set CUDNN_LIB_FOLDER="lib"
```

Reply:
+1 on @janeyx99 comment, please use lowercase "set"